_content/wg.md

```diff
@@ -96,6 +119,27 @@ The EICUG community currently supports and is using the following collaborative
 * [Zenodo documentation database for the EICUG](https://zenodo.org/communities/eicug/){:target="_blank"}.
 * GitHub Pages (for software code management and documentation)
 
-If you have questions or need assistance with the EIC Collaborative Tools please contact
-the coordinator: potekhin_at_bnl.gov.
+If you have questions or need assistance with the EIC Collaborative Tools, please contact the coordinator: <potekhin@bnl.gov> .
+
+<!-- [Contacting the conveners](eic-det2-conveners-l@lists.bnl.gov){:target="_blank"} 
+
+<span id="software"><br/></span>
+
+---
```

Comment:
Illegal double dash in comment. Ref. https://www.w3.org/TR/REC-xml/#sec-comments

Also, comment not closed.

Reply:
What are we going to do about this item?

Reply:
That span item... It's there for navigation? Was it removed? I think I put it there for anchor purposes a while ago.

Reply:
It belongs to the previous piece of text describing the old structure, that I commented and kept as history log. To my knowledge, it should break any anchor link from outside/inside this document.

Reply:
History shouldn't be kept as comments in the files. We use git because it keeps history for us.